Sources/Commands/SwiftTestTool.swift

```diff
@@ -219,7 +219,9 @@ public class SwiftTestTool: SwiftTool<TestToolOptions> {
                     processSet: processSet,
                     sanitizers: options.sanitizers,
```

Comment:
Since we're passing options, we can remove the `sanitizers` from the test runner.

Reply:
done